testplan/parser.py

```diff
@@ -159,10 +159,17 @@ def generate_parser(self):
             help='Directory path for XML reports.'
         )
 
+        report_group.add_argument(
+            '--http', dest='http_url',
+            default=None, metavar='URL',
+            help='Web URL for posting report.'
+        )
+
         report_group.add_argument(
             '--report-dir',
-            help='Target directory for tag filtered report output.',
-            default=defaults.REPORT_DIR, metavar='PATH')
+            default=defaults.REPORT_DIR, metavar='PATH',
+            help='Target directory for tag filtered report output.'
```

Comment:
Why are these lines changing? I cannot see a difference other than ordering. 
Please do not change more than is actually necessary in a PR review, lets keep PR's focused on the change at hand.